static/scss/partials/main.scss

```diff
@@ -323,9 +323,9 @@ button:active {
     max-width: $content-sm;
 }
 
-main,
-table {
+main, table {
     width: 100%;
+    transition: all .4s ease;
```

Comment:
Not too much of an issue, but [MDN recommends](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property):

> The set of properties that can be animated is subject to change. As such, you should avoid including any properties in the list that don't currently animate, as someday they might, causing unexpected results.

Reply:
This still applies :)